### DISCUSSION TITLE
Repo awsugid.github.io

### DISCUSSION THREAD

**Discussion by berviantoleo** [awsugid/forum#10] [Ideas]
Bagaimana kalau ada repo awsugid.github.io? Ready untuk membantu coding. 😄 

Tapi, kalau soal front-end saya biasa aja.

Insipirasi saya dari: https://awsusergroup.id/

Saya ada ide hanya untuk menyediakan info:

1. Daftar/List AWS UG di Indonesia
2. Mengarahkan ke forum ini jika ada pertanyaan, atau ide konten juga bisa (?)
3. Blog - ini opsional
4. Ide lainnya, jujur gak terlalu kepikiran,

**Comment by rioastamal** [comment]
Ide menarik. Statis website saja cukup saya rasa tidak perlu ada database, mungkin yang diperlukan adalah integrasi "webhook" antar layanan supaya dapat mentrigger commit kemudian nanti menjalankan CI/CD pipeline...

**Reply by berviantoleo** [reply]
Setuju, static web udah cukup.
